src/Api/NotificationCenter/Controllers/NotificationsController.cs

```diff
@@ -0,0 +1,73 @@
+﻿#nullable enable
+using Bit.Api.Models.Response;
+using Bit.Api.NotificationCenter.Models.Request;
+using Bit.Api.NotificationCenter.Models.Response;
+using Bit.Core.Models.Data;
+using Bit.Core.NotificationCenter.Commands.Interfaces;
+using Bit.Core.NotificationCenter.Models.Filter;
+using Bit.Core.NotificationCenter.Queries.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.NotificationCenter.Controllers;
+
+[Route("notifications")]
+[Authorize("Application")]
+public class NotificationsController : Controller
+{
+    private readonly IGetNotificationStatusDetailsForUserQuery _getNotificationStatusDetailsForUserQuery;
+    private readonly IMarkNotificationDeletedCommand _markNotificationDeletedCommand;
+    private readonly IMarkNotificationReadCommand _markNotificationReadCommand;
+
+    private const int DefaultPageSize = 10;
```

Comment:
```suggestion
    private const int _defaultPageSize = 10;
```

Our coding style calls for private variables to have an underscore.

https://github.com/bitwarden/server/blob/452a45b00b6781eb8310f3277274dfc801bcf152/.editorconfig#L54C1-L57C73

Reply:
💭  This will require a deployment if this paging size needs to be altered.

Should we expose an optional override via appsettings.json?

Reply:
Might be easier to expose pagesize to the request then.

Reply:
If we expose it in the request, let's make sure we have good validation logic with tight upper and lower bounds.

We don't want any potential bad implementations/clients abusing the endpoint.

I think for now having it as a private const in the controller is good.

Reply:
Fixed in ddd8192